banyand/query/processor.go

```diff
@@ -160,6 +163,53 @@ func (p *measureQueryProcessor) Rev(ctx context.Context, message bus.Message) (r
 		resp = bus.NewMessage(bus.MessageID(now), common.NewError("invalid event data type"))
 		return
 	}
+	resp = p.executeQuery(ctx, queryCriteria)
```

Comment:
We should increase the top N size by a factor of 2, as indicated in https://github.com/apache/skywalking/issues/13357#issuecomment-3036272668.

If the original topn size is 5, there should be 10.